composer/loggers/in_memory_logger.py

```diff
@@ -47,40 +46,40 @@ class InMemoryLogger(LoggerDestination):
             # which index in trainer.logger.destinations contains your desired logger.
             logged_data = trainer.logger.destinations[0].data
 
-    Args:
-        log_level (str | LogLevel, optional):
-            :class:`~.logger.LogLevel` (i.e. unit of resolution) at
-            which to record. Defaults to
-            :attr:`~.LogLevel.BATCH`, which records
-            everything.
-
     Attributes:
```

Comment:
separate PR: maybe rename `InMemoryLogger` -> `DictLogger`

Reply:
captured in CO-971